Libplanet.Net/Transports/BoundPeerExtensions.cs

```diff
@@ -76,8 +76,10 @@ internal static async Task<string> ResolveNetMQAddressAsync(this BoundPeer peer)
             // FIXME Dns.GetHostAddressesAsync() seems to always return at least one ip or throw
             // `SocketException`. if we can be sure, also should fix it to always assume there
             // is at least one instead of throwing additional `TransportException`.
-            string ip = addresses.FirstOrDefault()?.ToString()
+            // And, for now IPs are restricted with IPv4, and have to be fixed later on.
+            string ip = addresses.FirstOrDefault()?.MapToIPv4().ToString()
                 ?? throw new TransportException($"Failed to resolve for {addr}");
+            ip = (ip == "0.0.0.1") ? "127.0.0.1" : ip;
```

Comment:
```suggestion
            string ip = addresses.FirstOrDefault()?.MapToIPv4().ToString()
                ?? throw new TransportException($"Failed to resolve for {addr}");
            string ipv4 = (ip == "0.0.0.1") ? "127.0.0.1" : ip;
```
I don't have much good idea for using ipv4 only when localhost but IMO `ipv4` is the correct variable name in this case.

Reply:
Oh, I'm misreading it. 😓  just ignore this comment.